install.sh

```diff
@@ -20,6 +20,10 @@ sudo chown -R $webserver_user:$webserver_group var/logs var/cache
 rm -f web/index_dev.php
 rm -f web/app_dev.php
 
-echo now you need to create your database
-echo afterwards, run php bin/console migrations:migrate
+echo "generate ssh keys"
```

Comment:
ah, my bad, missed that in the command line - didn't even know that was possible :smile: